core-dev/packages/coq/coq.dev/opam

```diff
@@ -20,7 +20,7 @@ and homotopy type theory) and teaching.
 depends: [
   "ocaml" {>= "4.05.0"}
   "ocamlfind" {build}
-  "num"
+  "dune" {>= "2.5.1"}
```

Comment:
Would it make sense to mark this as `{build}` dependency?

Reply:
All `dune` dependencies lost their `{build}` annotation in the main opam-repository following problems after users downgraded their versions of Dune and got into a broken installation state (due to Dune installing `dune-package` files or something like that).

Reply:
Yes, this is a known problem, while it gets fixed by properly versioning things, dune has to be a non-build dep.